test/e2e/multi-stage.sh

```diff
@@ -20,6 +20,7 @@ os::cmd::expect_success "ci-operator --artifact-dir ${BASETMPDIR} --resolver-add
 UNRESOLVED_CONFIG="$( cat "${suite_dir}/config.yaml" )"
 export UNRESOLVED_CONFIG
 os::cmd::expect_success "ci-operator --artifact-dir ${BASETMPDIR} --resolver-address http://127.0.0.1:8080 --target with-references"
+os::cmd::expect_success "ci-operator --artifact-dir ${BASETMPDIR} --resolver-address http://127.0.0.1:8080 --target skip-on-success"
```

Comment:
Can we put this underneath or above the `UNRESOLVED_CONFIG` lines so it's not using that feature, as it's unrelated? You can pass the config literally with the flag like the other test cases do.